src/ansys/fluent/core/streaming_services/events_streaming.py

```diff
@@ -101,16 +127,17 @@ def register_callback(
         ------
         InvalidArgument
             If event name is not valid.
-        DisallowedValuesError
-            If an argument value not in the allowed values.
         """
         if event_name is None or callback is None:
             raise InvalidArgument("'event_name' and 'callback' ")
 
-        if event_name not in self.events_list:
-            raise DisallowedValuesError("event-name", event_name, self.events_list)
+        if type(event_name) == str:
+            warnings.warn(
```

Comment:
We can support both the enum and str as argument, so this warning is not needed.